### PR TITLE
Improve the 32bit experience when installing the 64bit agent.

### PR DESCRIPTION
**Pull request by jaffinito**
## Description

- Adds an action that will look for an existing install of the agent, capture its bitness and compare it to the installer's bitness.  IF they are different the installer throws an error message and forces an exit.
- The 64 bit installer now sets the `COR_PROFILER_PATH_32` and `CORECLR_PROFILER_PATH_32`  env vars in the registry for WAS and W3SVC pointing to the already installed 32bit profiler.  This will allow 32bit apps to get instrumented.

# Author Checklist
- [ ] Unit tests, Integration tests, and Unbounded tests completed
- [ ] Performance testing completed with satisfactory results (if required)

# Reviewer Checklist
- [ ] Perform code review
- [ ] Pull request was adequately tested (new/existing tests, performance tests)

### CODE REVIEW

File: src/Agent/MsiInstaller/Installer/Product.wxs [hunk 1]
```diff
@@ -809,13 +810,19 @@ SPDX-License-Identifier: Apache-2.0
           <MultiStringValue>COR_ENABLE_PROFILING=1</MultiStringValue>
           <MultiStringValue>COR_PROFILER={71DA0A04-7777-4EC6-9643-7D28B46A8A41}</MultiStringValue>
           <MultiStringValue>NEWRELIC_INSTALL_PATH=[NETAGENTFOLDER]</MultiStringValue>
+          <?ifdef IsWin64?>
+            <MultiStringValue>COR_PROFILER_PATH_32=[NETAGENT32FOLDER]NewRelic.Profiler.dll</MultiStringValue>
```
Comment (chynesNR) [inline src/Agent/MsiInstaller/Installer/Product.wxs:814]: You've verified that it works correctly if you have `COR_PROFILER_PATH` and `COR_PROFILER_PATH_32` set but not `COR_PROFILER_PATH_64`? I think it should, but the documentation is a little vague on how the priorities work.

Comment (jaffinito) [reply src/Agent/MsiInstaller/Installer/Product.wxs:814]: I read the docs and it looks like ti will work, but I will test.

Comment (jaffinito) [reply src/Agent/MsiInstaller/Installer/Product.wxs:814]: Yes, it works.

Comment (chynesNR) [reply src/Agent/MsiInstaller/Installer/Product.wxs:814]: Great, thanks!

**Review by chynesNR — APPROVED** [review]

**Review by nrcventura — APPROVED** [review]

**Pull request merged**